integration_test/tests/connection_tests.rs

```diff
@@ -21,4 +21,30 @@ mod integration {
         let conn_str = crate::common::generate_default_connection_str();
         env.connect_with_connection_string(&conn_str).unwrap();
     }
+
+    /**
+     * The following tests require a DSN called "ADF_Test" to be configured on the machine running the tests.
+     */
+
+    #[cfg(target_os = "windows")]
+    mod test_dsn {
+        use odbc::create_environment_v3;
```

Comment:
fyi: This odbc-rs crate does not have support for iODBC, so in my PR I added some helper functions to allocate the environment and connect.  Depending on which one is merged first, there will be some merge work with these tests.

Reply:
Sounds good to me. Whichever we go with it shouldn't be too hard to refactor these since it's a very basic connection!